docs/DeveloperGuide.md

```diff
@@ -257,47 +257,75 @@ _{Explain here how the data archiving feature will be implemented}_
+| `* * *`  | interior designer                         | add notes under a contact                                                            | keep track of the demands and requirements of each individual project                             |
+| `* * *`  | busy interior designer                    | add a deadline under a contact                                                       | keep track of my deadlines easily                                                                 |
+| `* * *`  | user                                      | favourite certain contacts                                                           | look up favourites and contact them easily                                                        |
+| `* *`    | interior designer                         | check upcoming deadlines in chronological order                                      | keep track of deadlines automatically                                                             |
+| `* *`    | long-time user                            | separate past and current clients                                                    | avoid contacting clients with similar names or old clients that I am not presently working with   |
+| `* *`    | new user                                  | view a list of commands                                                              | know what commands are available and the right commands to use                                    |
+| `*`      | interior designer                         | add images under a client                                                            | keep track of images such as floor plans and inspirational designs relevant to the client         |
+| `*`      | interior designer                         | add a high importance tag to a client                                                | take note of pressing issues regarding a client, such as mobility issues                          |
+| `*`      | interior designer                         | track and calculate costs accumulated for a client                                   | at a glance, know how much money has been spent on them for a project                             |
+| `*`      | interior designer                         | generate invoices                                                                    | easily generate, store and print invoices for my clients                                          |
+| `*`      | interior designer                         | send out automated messages/emails to clients to wish them well on festive occasions | maintain good rapport with clients                                                                |
+| `*`      | user                                      | create contacts by importing information from other apps                             | create contacts more easily                                                                       |
 
 *{More to be added}*
```

Comment:
Should we remove this line? Looks to me like it was meant to be a prompt for us to fill up the blanks 🤔

Reply:
I left it in because I think we will add more if we find ourselves free enough to add more new features, but yea, I don't think it's necessary to keep it in.